packages/vertexai/src/requests/request.ts

```diff
@@ -134,6 +133,7 @@ export async function makeRequest(
 ): Promise<Response> {
   const url = new RequestUrl(model, task, apiSettings, stream, requestOptions);
   let response;
+  let fetchTimeoutId: string | number | NodeJS.Timeout | undefined;
```

Comment:
This Node type disappears after compiling, right? Doesn't cause any issues in browers?

Reply:
Yeah this type would only be used internally. It's not in our bundles and not in the public types.